src/lib/shadcnComponents/ui/separator/index.ts

```diff
@@ -0,0 +1,7 @@
+import Root from './separator.svelte';
+
+export {
+  Root,
+  //
+  Root as Separator,
```

Comment:
why export both?

Reply:
Shadcn automatically has both the unnamed and renamed version in all index.ts files. This is how it is in their docs.